### ISSUE TITLE
Syntax errors in sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90

### ISSUE THREAD

**Issue by DusanJovic-NOAA** [NOAA-EMC/UFS_UTILS#147]
Syntax errors in `sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90` with gnu compilers:

```
/home/dusan/simple-ufs/src/preproc/sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90:580:132:

  580 |       write(string,'(a,es8.1)') 'land_frac and lake_frac are adjusted such that their sum is 1 at points where inland=1; land_frac cutoff is',land_cutoff
      |                                                                                                                                    1
Error: Line truncated at (1) [-Werror=line-truncation]
/home/dusan/simple-ufs/src/preproc/sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90:580:33:

  580 |       write(string,'(a,es8.1)') 'land_frac and lake_frac are adjusted such that their sum is 1 at points where inland=1; land_frac cutoff is',land_cutoff
      |                                 1
Error: Unterminated character constant beginning at (1)
/home/dusan/simple-ufs/src/preproc/sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90:776:132:

  776 |     write(string,'(a,es8.1)') 'land_frac is adjusted to 1-lake_frac where lake_frac>0 but left unchanged where lake_frac=0. This could lead to land_frac+lake_frac<1 at some inland points; land_frac cutoff is',land_cutoff
      |                                                                                                                                    1
Error: Line truncated at (1) [-Werror=line-truncation]
/home/dusan/simple-ufs/src/preproc/sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90:776:31:

  776 |     write(string,'(a,es8.1)') 'land_frac is adjusted to 1-lake_frac where lake_frac>0 but left unchanged where lake_frac=0. This could lead to land_frac+lake_frac<1 at some inland points; land_frac cutoff is',land_cutoff
      |                               1
Error: Unterminated character constant beginning at (1)
```
and
```
/home/dusan/simple-ufs/src/preproc/sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90:275:14:

  275 |           IF (two_section == .false.) THEN
      |              1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/home/dusan/simple-ufs/src/preproc/sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90:279:17:

  279 |               IF(enclosure_cnvx(v, 4, p, co_gc) == .true.) THEN
      |                 1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/home/dusan/simple-ufs/src/preproc/sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90:299:17:

  299 |               IF(enclosure_cnvx(v, 4, p, co_gc) == .true.) THEN
      |                 1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/home/dusan/simple-ufs/src/preproc/sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90:318:17:

  318 |               IF(enclosure_cnvx(v, 4, p, co_gc) == .true.) THEN
      |                 1
Error: Logicals at (1) must be compared with .eqv. instead of ==

```

**Comment by GeorgeGayno-NOAA** [comment]
Dusan made these changes while testing the mpi abort branch.  So I will include them under feature/mpi_abort and #140.

**Comment by GeorgeGayno-NOAA** [comment]
Merged to 'develop' at d29cc35.  Closing ticket.
